media-api/app/lib/elasticsearch/sorts.scala

```diff
@@ -38,14 +68,17 @@ object sorts {
     builder
   }
 
+  val DescField = "-(.+)".r
   type Field = String
+  case class SortParams(field: Field, order: SortOrder)
 
-  val DescField = "-(.+)".r
+  def parseSort(sortBy: Option[String]): SortParams = {
+    val sorts = sortBy.map {
+      case DescField(field) => SortParams(field, SortOrder.DESC)
+      case field            => SortParams(field, SortOrder.ASC)
+    }.getOrElse(SortParams("uploadTime", SortOrder.DESC))
 
-  def parseSorts(sortBy: String): Seq[(Field, SortOrder)] =
```

Comment:
`parseSort` no longer takes multiple sorts but we didn't seem to use it anywhere